data/trie/patriciaMerkleTrie.go

```diff
@@ -163,13 +163,23 @@ func (tr *patriciaMerkleTrie) VerifyProof(proofs [][]byte, key []byte) (bool, er
 	tr.mutOperation.RLock()
 	defer tr.mutOperation.RUnlock()
 
+	wantHash, err := tr.Root()
```

Comment:
There is no unit test for err here.

Reply:
tr.Root() is tested elsewhere, and if the wantHash is wrong, some tests will fail.